python/pyhail/dataset.py

```diff
@@ -1010,10 +1010,123 @@ def sparkinfo(self):
+        g. For example, the PL for (A, T) at 0/1 is the minimum of the
+        PLs for 0/1 (50) and 1/2 (45), and thus 45.
+
+        Fixing an alternate allele and biallelic variant, downcoding
+        gives a map from multiallelic to biallelic alleles and
+        genotypes. The biallelic AD entry for an allele is just the
+        sum of the multiallelic AD entries for alleles that map to
+        that allele. Similarly, the biallelic PL entry for a genotype
+        is the minimum over multiallelic PL entries for genotypes that
+        map to that genotype.
+
+        By default, GQ is recomputed from PL. If ``propagate_gq=True``
+        is passed, the biallelic GQ field is simply the multiallelic
+        GQ field, that is, genotype qualities are unchanged.
+
+        Here is a second example for a het non-ref::
+
+          A   C,T 1/2:2,8,6:16:45:99,50,99,45,0,99
+
+        splits as::
+
+          A   C   0/1:8,8:16:45:45,0,99
+          A   T   0/1:10,6:16:50:50,0,99
+
+        **VCF Info Fields**
+
+        Hail does not split annotations in the info field. This means
+        that if a multiallelic site with ``info.AC`` value ``[10, 2]`` is
+        split, each split site will contain the same array ``[10,
+        2]``. The provided allele index annotation ``va.aIndex`` can be used
+        to select the value corresponding to the split allele's
+        position:
+
+        >>> (hc.import_vcf('data/sample.vcf')
+        >>>  .split_multi()
+        >>>  .filter_variants_expr('va.info.AC[va.aIndex - 1] < 10', keep = False))
+
+        VCFs split by Hail and exported to new VCFs may be
+        incompatible with other tools, if action is not taken
+        first. Since the "Number" of the arrays in split multiallelic
+        sites no longer matches the structure on import ("A" for 1 per
+        allele, for example), Hail will export these fields with
+        number ".".
+
+        If the desired output is one value per site, then it is
+        possible to use annotatevariants expr to remap these
+        values. Here is an example:
+
+        >>> (hc.import_vcf('data/sample.vcf')
+        >>>  .split_multi()
+        >>>  .annotate_variants_expr('va.info.AC = va.info.AC[va.aIndex - 1]')
+        >>>  .export_vcf('data/export.vcf'))
+
+        The info field AC in ``data/export.vcf`` will have ``Number=1``.
+
+        **Annotations**
+
+        ``split_multi`` adds the following annotations:
+
+         - **va.wasSplit** (*Boolean*) -- true if this variant was
+           originally multiallelic, otherwise false.
+
+         - **va.aIndex** (*Int*) -- The original index of this
+           alternate allele in the multiallelic representation (NB: 1
+           is the first alternate allele or the only alternate allele
+           in a biallelic variant). For example, 1:100:A:T,C splits
+           into two variants: 1:100:A:T with ``aIndex = 1`` and
+           1:100:A:C with ``aIndex = 2``.
```

Comment:
The bulleted list is in a blockquote that is picking style: border-left: 0.5em #eee solid; and that's causing it.  In our style.css, we have:

```
blockquote {
  color: #666666;
  margin: 0;
  padding-left: 3em;
  border-left: 0.5em #EEE solid;
}
```

Not sure what the implications of altering this would be, I'll leave it to you.  I think the doc markup is OK.